.github/workflows/prerelease.yml

```diff
@@ -80,12 +78,28 @@ jobs:
           sudo mv /usr/bin/python3 /usr/bin/python3-default
           sudo ln -s ${{ env.pythonLocation }}/bin/python3 \
           /usr/bin/python3
+      - name: Install process user data file script dependencies
+        run: python -m pip install boto3 jinja2
+      - name: Process user data file
+        id: process-user-data
+        # Process the user data script with the Windows admin password
+        # pulled from AWS Parameter Store. Return the admin password to
+        # pass into Packer as an environment variable.
+        run: |
+          output=$(python .github/scripts/process_user_data_file.py)
+          echo "::add-mask::$output"
+          echo "::set-output name=pass::$output"
       - name: Create machine image
         env:
           GITHUB_IS_PRERELEASE: ${{ github.event.release.prerelease }}
           GITHUB_RELEASE_TAG: ${{ github.event.release.tag_name }}
           GITHUB_RELEASE_URL: ${{ github.event.release.html_url }}
+        # See https://www.packer.io/guides/hcl/variables under the
+        # "From environment variables" section for details about
+        # PKR_VAR environment variables
         run: |
+          PKR_VAR_winrm_password=\
```

Comment:
Same change as in `build.yml` but the variable should be added alphabetically.

Reply:
@mcdonnnj please refer to this comment: https://github.com/cisagov/windows-server-packer/pull/10#discussion_r876575860

Reply:
@itsmostafa Please refer to this Action run where my suggestion works without issue: https://github.com/cisagov/windows-server-packer/actions/runs/2416744567

Reply:
hmm, @mcdonnnj not sure why this didnt work previously. Just resolved this by committed your suggestion here: [475badb](https://github.com/cisagov/windows-server-packer/pull/10/commits/475badbcae3babdbac3f53b94ba73f31c99237cd)

Reply:
Looking at the logs from the Action run that failed I think the way you formatted it ran afoul of Packer's argument parsing.